src/chapters/razor-views/studio.rst

```diff
@@ -0,0 +1,120 @@
+
+In ``Views/Spa``, we have a Razor template called ``Menu``.
+Inside ``Menu.cshtml``, there are two ``div`` elements.
+Let's add some children to the ``div`` with the id, ``clientProfile``.
+
+#. Add an ``h3`` that says "Client profile".
+#. Next, display the value of the ``name`` parameter from the form. 
+
+   .. admonition:: Tip
+
+      This is a two step process. 
+
+      #. In the controller, add a ``ViewBag`` property to hold the ``name`` value.
+      #. In the view, use that ``ViewBag`` property to display the ``name`` in a ``p`` tag. 
+
+Run the application and head to ``localhost:5001/spa`` to see the results. When we
+fill out the form, we should see a new page with the client profile heading and name 
+at the top.
+
+List All Appropriate Facial Treatments
+--------------------------------------
+
+To provide treatment suggestions, ``SpaController.Menu()`` uses the ``CheckSkinType()`` method
+and fills a list with facial treatments that benefit the user's skin type. Now, we just need to use Razor to
+display the contents of the ``appropriateFacials`` list.
+
+#. Add the client's ``skintype`` and the ``appropriateFacials`` list as a ``ViewBag`` property.
+
+#. Now, head back to ``Menu.cshtml`` and checkout the empty ``div`` with the id,
+   ``servicesMenu``. 
+
+#. Pass in the ``skintype`` variable to the ``<p>`` tag.
+
+#. Iteratively add the values in ``appropriateFacials`` to an unordered list. If you need a quick reminder of
+   the syntax, review the :ref:`@foreach section <razor-foreach>` .
+
+Mani or Pedi?
+-------------
+
+We want to display a description for the nail service the user selects.
+Inside the ``servicesMenu`` div, use ``@if`` to determine if the value of 
+``manipedi`` is a ``"manicure"`` or ``"pedicure"``.
+
+#. If the value of ``manipedi`` is ``"manicure"``, display this description:
+
+      Our manicure is a great way to spend 30 minutes of your day! We use shea butter hand cream and the finest gel polish.
+
+#. If the value of ``manipedi`` is ``"pedicure"``, display this description:
+
+      Relax for 45 minutes in pure luxury! Our massage chairs and experienced nail techs are here to get your feet in shape for sandal season!
+
+
+End Result
+----------
+
+After you are done with the studio, you should be able to fill out the form,
+click "Submit", and see your profile page.
+
+.. figure:: figures/completedform.png
+   :scale: 40%
+   :alt: Form completed with the name, "Yolanda", combination skin and a preference for pedicures.
+
+   A client completes the spa day form.
+
+.. figure:: figures/endresultprofilepage.png
+   :scale: 40%
+   :alt: Profile showing Yolanda's information, recommended facial treatments, and pedicure description.
+
+   The client's service menu.
+
+Bonus Mission
+-------------
+
+#. Modify ``site.css`` to get some CSS practice. Try add a footer with square
+   shaped ``div`` elements. Each square should be a different color for
+   different available nail polishes.
```

Comment:
Since ``site.css`` has a lot of CSS that Microsoft in it, I might add a suggestion as to where to put this. I think some students would like this challenge but might be deterred from trying it because of how much is already going on in ``site.css``.